### PR TITLE
Add com.github.yucefsourani.albasheer-electronic-quran-browser

### PR DESCRIPTION
**Pull request by yucefsourani**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by barthalion** [comment]
bot, build com.github.yucefsourani.albasheer-electronic-quran-browser

**Comment by yucefsourani** [comment]
Hi, I got an error  in "Check for 128x128 icon" .

I don't remember reading about this in help pages.

I have added an icon 128x128 , Can we try test building again?

Thank you.

**Comment by bilelmoussaoui** [comment]
bot, build com.github.yucefsourani.albasheer-electronic-quran-browser

You can trigger a build as well

File: com.github.yucefsourani.albasheer-electronic-quran-browser.json [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+                }
+        },
+        "sources": [
+            {
+                "type": "git",
+                "url": "https://github.com/yucefsourani/albasheer-electronic-quran-browser"
+        },
+        {
+                "type": "file",
+                "path": "com.github.yucefsourani.albasheer-electronic-quran-browser.appdata.xml"
```
Comment (LeandroStanger) [inline com.github.yucefsourani.albasheer-electronic-quran-browser.json:74]: New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html

**Comment by yucefsourani** [comment]
bot, build com.github.yucefsourani.albasheer-electronic-quran-browser

**Comment by yucefsourani** [comment]
bot, build com.github.yucefsourani.albasheer-electronic-quran-browser

**Pull request closed without merging**